vita3k/shader/src/translator/data.cpp

```diff
@@ -561,7 +561,7 @@ bool USSETranslatorVisitor::vldst(
 
     Operand to_store;
 
-    if (m_program.is_secondary_program_available()) {
+    if (is_translating_secondary_program()) {
```

Comment:
SA is not writeable on primary program. So i'm sure this is mistake.

Reply:
:+1: